client/layout/index.jsx

```diff
@@ -427,7 +426,9 @@ class Layout extends Component {
 				{ config.isEnabled( 'legal-updates-banner' ) && (
 					<AsyncLoad require="calypso/blocks/legal-updates-banner" placeholder={ null } />
 				) }
-				{ ! isA8CForAgencies() && <GlobalNotifications /> }
+				{ config.isEnabled( 'layout/global-notifications' ) && (
+					<AsyncLoad require="calypso/layout/global-notifications" placeholder={ null } />
```

Comment:
Aren't we already async-loading most of it (`client/notifications`) right inside this module? 

Also, I guess there might be unexpected consequences due to the corresponding style being lazy-loaded.

Reply:
Good point, I didn't notice there is another async load inside `GlobalNotifications`. I changed it back to synchronous loading. The ICFY stats say that mere 185 bytes were removed from the entrypoint.